src/models/create.ts

```diff
@@ -8,7 +8,7 @@
 export interface ICreateProposalRequest {
     partialTx: string;
     authPassword: string;
-    expiresAt?: Date;
+    expiresAt?: string;
```

Comment:
When dealing directly with HTTP requests it is better not to try and convert the data type on the model.

Instead, it will be processed and converted to `Date` at the code and used internally.